app/controllers/items_controller.rb

```diff
@@ -16,8 +26,15 @@ def edit
    def update
       @merchant = Merchant.find(params[:merchant_id])
       @item = Item.find(params[:item_id])
-      # require 'pry';binding.pry
-      if @item.update(item_params)
+
+      if params[:commit] == 'Enable'
```

Comment:
We'll just need to be thoughtful about when we merge your PR and mine because I don't want the strong params I have in mine to overwrite your conditions or vice versa